py-polars/polars/series/series.py

```diff
@@ -1021,6 +1021,7 @@ def _arithmetic(self, other: Any, op_s: str, op_ffi: str) -> Self:
         else:
             other = maybe_cast(other, self.dtype)
             f = get_ffi_func(op_ffi, self.dtype, self._s)
+            print(f)
```

Comment:
I'm guessing this one got in by mistake? :)

Reply:
Oei.. yes.